pkg/loghttp/series_test.go

```diff
@@ -41,6 +41,17 @@ func TestParseSeriesQuery(t *testing.T) {
 			false,
 			mkSeriesRequest(t, "1000", "2000", []string{`{a="1"}`, `{b="2", c=~"3", d!="4"}`}),
 		},
+		{
+			"accept match[]",
+			withForm(url.Values{
+				"start":   []string{"1000"},
+				"end":     []string{"2000"},
+				"match":   []string{`{a="1"}`},
+				"match[]": []string{`{b="2", c=~"3", d!="4"}`},
```

Comment:
Can you add a test with multiple string value from `match[]`.

Just to ensure we support this use case where multiple matches are provided using that new `match[]`.

Like this 

```
curl -g 'http://localhost:9090/api/v1/series?' --data-urlencode 'match[]=up' --data-urlencode 'match[]=process_start_time_seconds{job="prometheus"}'
```

I think you can do that simply by adding to the array:

```
"match[]": []string{`{b="2", c=~"3", d!="4"}`,`{a="1"}`},
```

Reply:
Sure